src/FACT_internal.c

```diff
@@ -1146,18 +1146,19 @@ void FACT_INTERNAL_ActivateEvent(
 			/* FIXME: Incorporate 2nd derivative into the interpolated pitch */
 			skipLoopCheck = elapsed <= (evtInst->timestamp + evt->value.ramp.duration);
 			svResult = (
-				evt->value.ramp.initialSlope *
-				evt->value.ramp.duration *
+				evt->value.ramp.initialSlope * 100 *
+				evt->value.ramp.duration / 1000 *
 				10 /* "Slices" */
```

Comment:
All this unit adjustment should be re-worked, but this is the conversions that make may pitch ramp tests pass.